### DISCUSSION TITLE
HomeKit Secure Video?

### DISCUSSION THREAD

**Discussion by dallombra** [homebridge-simplisafe3/homebridge-simplisafe3#289] [Feature Requests]
Hello! Avid user of this plugin, so let me start by saying Great Work!

Now to the question/idea: recently Homebridge implemented compatibility with HomeKit secure video. Do you have any plans to implement this in your plugin?

Many thanks!

**Comment by shamoon** [comment]
Hi @dallombra Yea, currently not planned but I have been looking into it. Still not clear if its quite possible with SS video setup...

**Reply by Team-NinjaDev** [reply]
I really hope that it is and love this plugin

**Reply by dallombra** [reply]
Thanks for the feedback! If you need testers, I’d be happy to help!

**Reply by DebasishBahinipati** [reply]
This will be a great feature, I am willing to test as well. Also if we can add some options to store the motion snapshots for the video doorbell, that will be great as well in the mean time (probably to icloud or somewhere).
